tabnames.py

```diff
@@ -0,0 +1,35 @@
+import mysqlhack
+import org.bukkit as bukkit
+import json
+from java.util import UUID as UUID
+from helpers import *
+from org.bukkit import *
+from traceback import format_exc as trace
+from iptracker_secrets import *
+
+tabnames_version = "v1.0.0"

```

Comment:
I wasn't part of that discussion... Doesn't make a lot of sense to have **internal** version dependencies, i.e. inside one single repository. We can clean up these things in [toolbox](https://github.com/RedstonerServer/toolbox) then.

Anyway, this variable isn't used so it's quite useless.